### DISCUSSION TITLE
Exclude days/times from healthcheck?

### DISCUSSION THREAD

**Discussion by 2sls** [healthchecks/healthchecks#677] [Q&A]
I have a script that I only care about running on weekdays and was wondering if there's a way to have Healthcheck not expect a ping on weekends.

I know I can write into my script to ping Healthcheck regardless but was wondering if there was a more elegant way to solve this problem.

**Comment by cuu508** [comment]
In the dialog where you configure the period and grace time, select the "cron" option and use a cron schedule:

`* * * * MON-FRI` – expects a ping once a minute but only on weekdays

`0 * * * MON-FRI` – expects a ping at the start of every hour but only on weekdays

`0 0 * * MON-FRI` – expects a ping at midnight but only on weekdays

[cron syntax reference](https://healthchecks.io/docs/cron/)

**Comment by 2sls** [comment]
Thanks - worked perfectly!
